features/logging/cluster_logging_operator.feature

```diff
@@ -152,3 +152,64 @@ Feature: cluster-logging-operator related test
       | card_name | #{cb.card} |
     Then the step should succeed
     """
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-33868
+  @admin
+  @destructive
+  Scenario: Expose more fluentd knobs to support optimizing fluentd for different environments - Invalid Values
+    Given I obtain test data file "logging/clusterlogging/cl_fluentd-buffer_Invalid.yaml"
+    When I run the :create client command with:
+      | f | cl_fluentd-buffer_Invalid.yaml |
+    Then the step should fail
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-33868
+  # @case_id OCP-33866
```

Comment:
@gkarager please don't put 2 test cases in one scenario, I suggest to merge these 2 cases into 1 case in Polarion.

Reply:
@QiaolingTang Done.